cypress/integration/dib-travel/company-settings/lodge-cards.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { PaymentMethod } from '../../../models';
+
+describe('Company Settings - Payment Method - Lodge Cards', () => {
+  let paymentMethodForm: PaymentMethod;
+
+  before(() => {
+    cy.fixture('company-settings/payment-method-form').then((paymentMethodFormDetails) => {
+      paymentMethodForm = paymentMethodFormDetails;
+    });
+  });
+
+  beforeEach(() => {
+    cy.loginAgent();
+    cy.visit('/company-management/payment-method/lodge-cards');
+  });
+  //TODO: Cypress cannot see any elements on this page and all tests below failed.
+  it('should display company settings/payment method in navbar menu', () => {
+    cy.get('dib-navbar dib-hamburger-icon').click();
+    cy.get('.cdk-overlay-container dib-navbar-panel').contains(' Payment Method ');
+  });
+
+  it('submits empty form for creating lodge card', () => {
+    cy.get('dib-company-management dib-payment-method dib-lodge-cards ui-button[type=primary]').click();
+    cy.get('.cdk-overlay-container dib-lodge-card-dialog ui-button[type=success]').click();
+    cy.get('.cdk-overlay-container dib-lodge-card-dialog').should('be.visible');
+  });
+
+  it('closes dialog for creating lodge card', () => {
+    cy.get('dib-company-management dib-payment-method dib-lodge-cards ui-button[type=primary]').click();
+    cy.get('.cdk-overlay-container dib-lodge-card-dialog ui-button button').contains(' Cancel ').click();
+    cy.get('dib-company-management dib-payment-method dib-lodge-cards h1').should('contain', ' Lodge Cards ');
+  });
+
+  it('creates a new lodge card', () => {
```

Comment:
This test is failed on my side. Please revisit.

```
cy.type() failed because it targeted a disabled element.

The element typed into was:

  > <input _ngcontent-hcn-c3="" class="input" maxlength="999" name="email" autocomplete="on" type="email">

Ensure the element does not have an attribute named disabled before typing into it.
````